website/blog/2023-08-14-aa-compliance-accessibility.md

```diff
@@ -0,0 +1,145 @@
+Used ARIA roles to convey important information to assistive technologies.
+
+**Panel, Tile, Accordion, Tabs, Dialog, Wizard, Media**:
+Ensured that all interactive elements within these widgets are keyboard accessible.
+Provided proper labeling and ARIA attributes for enhanced screen reader support.
+
+
+### Different tools used to test Accessibility
+In our quest to ensure accessibility compliance across our Wavemaker’s diverse range of widgets, we employ various tools and techniques to test and validate the accessibility of each element. Below are some of the tools we leverage to ensure that every widget on our platform is accessible to all users:
+
+**Screen Readers (e.g., NVDA/AssistiveLabs and VoiceOver)**: We test all widgets using popular screen readers like NVDA (NonVisual Desktop Access) and VoiceOver to experience the website as users with visual impairments would. This helps us identify issues and optimize our widgets for smooth navigation and comprehension.
+
+**Keyboard Navigation Testing**: We conduct extensive keyboard navigation testing to ensure that users can access and interact with all widgets using only the keyboard, without relying on a mouse.
+
+**ARIA Validation Tools**: We use ARIA (Accessible Rich Internet Applications) validation tools to verify the appropriate use of ARIA attributes and roles in our widgets, enhancing their compatibility with assistive technologies.
+
+**Form Accessibility Testing**: Our forms undergo rigorous accessibility testing to ensure they are easily navigable and usable with screen readers, providing descriptive labels, and handling errors effectively.
+
+**Automated Accessibility Testing Tools (e.g., Axe, Pa11y)**: We utilize automated testing tools like Axe and Pa11y to scan our widgets and identify potential accessibility issues. These tools help us streamline the testing process and catch common problems quickly.
+
+**Browser Developer Tools**: The built-in accessibility inspection tools within modern web browsers assist us in identifying and rectifying accessibility issues for each widget.
+
+**User Testing and Feedback**: Real-world testing by users with disabilities is a crucial aspect of our accessibility evaluation process. We seek feedback from diverse user groups to gain valuable insights and address any usability challenges.
+
+**Continuous Monitoring**: We continuously monitor our Wavemaker’s accessibility to ensure that any new updates or content additions maintain the same high level of accessibility compliance.
+
+By combining these testing methods and tools, we can confidently offer a Wavemaker with widgets that are accessible, inclusive, and easy to use for all users, regardless of their abilities or assistive technologies. Our commitment to accessibility remains an ongoing journey, and we are dedicated to providing an inclusive experience for every visitor to our website built through our platform.
```

Comment:
Again close saying -- "we offer excellent platform to build rich web applications that are accessible..." remove we can confidently offer a Wavemaker with widgets...

Reply:
This is done